src/analysis/complex/upper_half_plane.lean

```diff
@@ -84,79 +88,149 @@ begin
   fin_cases i; assumption,
 end
 
-lemma denom_ne_zero (g : SL(2, ℝ)) (z : ℍ) : denom g z ≠ 0 :=
-linear_ne_zero (↑ₘg 1) z (g.row_ne_zero 1)
+lemma denom_ne_zero (g : GL(2, ℝ)⁺) (z : ℍ) : denom g z ≠ 0 :=
+begin
+  intro H,
+  have DET := (mem_GL_pos _).1 g.property,
+  have hz := z.property,
+  simp only [subtype.val_eq_coe, general_linear_group.coe_det_apply] at DET,
```

Comment:
If you use `.prop` instead of `.property` you don't need `subtype.val_eq_coe`.

```suggestion
  have DET := (mem_GL_pos _).1 g.prop,
  have hz := z.prop,
  simp only [general_linear_group.coe_det_apply] at DET,
```